api/src/main/java/ca/bc/gov/educ/api/batchgraduation/reader/DeleteStudentReportsPartitioner.java

```diff
@@ -48,13 +48,13 @@ public Map<String, ExecutionContext> partition(int gridSize) {
         logger.debug("Filter Schools for deleting student reports");
         boolean processAllReports = "ALL".equalsIgnoreCase(searchRequest.getActivityCode());
         Long batchId = jobExecution.getId();;
```

Comment:
I know it's not your code, but please remove double ;; on line 50.

Reply:
I will remove the double.

Reply:
Done